website/cue/reference/components/sources/host_metrics.cue

```diff
@@ -430,7 +430,7 @@ components: sources: host_metrics: {
 				mode: {
 					description: "Which mode the CPU was running in during the given time."
 					required:    true
-					examples: ["idle", "system", "user", "nice"]
+					examples: ["idle", "system", "user", "nice", "io-wait"]
```

Comment:
Is this a typo? The code says `io_wait` here (underscore vs hyphen).

Reply:
Fixed typo, thanks for noticing it!